modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/resolver/RequiredParameterValueResolverWrapperTestCase.java

```diff
@@ -78,6 +81,7 @@ public void nonNullValue() throws Exception {
   }
 
   @Test
+  // This cannot be validated at the AST because the exception results of a provided expression evaluating to null
```

Comment:
use appropriate comment format for a method or add this this comment in the method.